src/locale/deviceLocales.ts

```diff
@@ -41,17 +41,23 @@ export function getLocales() {
     }
 
     if (typeof locale.languageTag === 'string') {
-      if (locale.languageTag.startsWith('zh-Hans')) {
-        // Simplified Chinese to zh-CN
-        locale.languageTag = 'zh-CN'
+      if (
+        locale.languageTag === 'zh-CN' ||
+        locale.languageTag.startsWith('zh-Hans')
+      ) {
+        // Simplified Chinese to zh-Hans
+        locale.languageTag = 'zh-Hans'
       }
-      if (locale.languageTag.startsWith('zh-Hant')) {
-        // Traditional Chinese to zh-TW
-        locale.languageTag = 'zh-TW'
+      if (
+        locale.languageTag === 'zh-TW' ||
+        locale.languageTag.startsWith('zh-Hant')
+      ) {
+        // Traditional Chinese to zh-Hant
+        locale.languageTag = 'zh-Hant'
       }
       if (locale.languageTag.startsWith('yue')) {
-        // Cantonese (Yue) to zh-HK
-        locale.languageTag = 'zh-HK'
+        // Cantonese (Yue) to yue
+        locale.languageTag = 'yue'
```

Comment:
To match the two variants on Android and iOS:
`yue-Hans` and `yue-Hant`.

Not using `yue-Hant` directly due to poor browser compatibility.